src/herder/test/TransactionQueueTests.cpp

```diff
@@ -1440,12 +1440,15 @@ TEST_CASE("transaction queue starting sequence boundary",
             auto lcl = app->getLedgerManager().getLastClosedLedgerHeader();
             lcl.header.ledgerSeq = ledgerSeq;
             auto txSet = tq.toTxSet(lcl);
-            REQUIRE(txSet->mTransactions.size() == size);
+            REQUIRE(txSet->sizeTx() == size);
+            /* This is no longer valid. The TxSet is always in Hash order, the
+            original queuing order is no longer preserved.
```

Comment:
I'll remove this block once getting sign-off.

Reply:
This part should still be removed. TxSet constructed from the `TransactionQueue` is hash sorted, so the original indices won't work.

Reply:
Fixed. Although I stared at this test for a long time and still not 100% clear what it's trying to do.